source/backend/dal/Repositories/LeaseRepository.cs

```diff
@@ -188,6 +188,124 @@ public PimsLease Get(long id)
             return lease;
         }
 
+        public PimsLease GetNoTracking(long id)
+        {
+            this.User.ThrowIfNotAuthorized(Permissions.LeaseView);
+
+            PimsLease lease = this.Context.PimsLeases.AsSplitQuery()
+                .Include(l => l.PimsPropertyLeases)
+                .ThenInclude(p => p.Property)
+                    .ThenInclude(p => p.Address)
+                    .ThenInclude(p => p.Country)
+                .Include(l => l.PimsPropertyLeases)
+                    .ThenInclude(p => p.Property)
```

Comment:
question - what's the difference with the **Get** method? I see no "AsNoTracking()" here. 
Am I missing something?

Reply:
Just asNoTracking() is the only change. I tried adding asNoTracking to the pre-existing method but immediately saw two workflows immediately start to fail so unable to change the existing method, however I need asNoTracking otherwise I am unable to delete the related lease entities due to multiple tracked entity changes. I'll add a TODO here because ideally this would be fixed in the long term.